spec/manuals_publisher/removing_content_spec.rb

```diff
@@ -0,0 +1,59 @@
+feature "Removing content on Manuals Publisher", manuals_publisher: true do
+  include ManualsHelpers
+
+  let(:title) { title_with_timestamp }
+  let(:section_title) { title_with_timestamp }
+  let(:section_slug) { section_title.downcase.tr(" ", "-").gsub(/[^\w-]/, '') }
+
+  scenario "Remove a section from a published manual" do
+    given_there_is_a_published_manual_with_sections
+    when_i_remove_a_section_and_ignore_the_500
```

Comment:
Only minor but I think we maybe introduce some confusion with this method name but then no other mention of the 500 in the code. I'd suggest we drop this from the name and make a comment in the method.